app.js

```diff
@@ -276,16 +302,24 @@ const tslint = {
     printMsg('Creating the Angular App ...');
     cp.execSync(`${angularNewCommand} > ${toNull}`);
   } else {
-    printMsg('Installing NPX...');
-    cp.execSync(`npm install -g npx > ${toNull}`);
-    printDone('Installing NPX...');
-    printMsg('Creating the Angular App ...');
-    cp.execSync(
-      `npx -p @angular/cli@${answers.version} ${angularNewCommand} > ${toNull}`,
-    );
+    let version;
+    try {
+      version = cp.execSync(`npx -version`);
+    } catch (error) {
+      printMsg('Installing NPX...');
+      cp.execSync(`npm install -g npx > ${toNull}`);
+      printDone('Installing NPX...');
+    }
+    if (version && !!version.toString()) {
```

Comment:
Since version will be converted to a string, the !!version.toString() becomes redundant